src/openlmis-quantity-unit-input/openlmis-quantity-unit-input.html

```diff
@@ -0,0 +1,27 @@
+<div ng-if="vm.showInDoses" >
+    <input id="quantity"                                       
+        class="form-control"
+        ng-class="vm.inputClass"
+        ng-model="vm.item.quantity"
+        ng-change="vm.changeValue(vm.item)"
+        required positive-integer />
+</div>
+<div class="openlmis-input" ng-if="!vm.showInDoses">
+    <input id="quantityInPacks"                                 
+        class="form-control"
+        ng-class="vm.inputClass"
+        ng-model="vm.item.quantityInPacks"
+        ng-change="vm.changeValue(vm.item)"
+        placeholder="{{'openlmisInputDosesPacks.Packs' | message }}"
+        required positive-integer />
+    <p> ( + </p>
+    <input id="quantityRemainderInDoses"  
+        class="form-control"
+        ng-class="vm.inputClass"
+        ng-model="vm.item.quantityRemainderInDoses"
+        ng-change="vm.changeValue(vm.item)"
+        ng-disabled="vm.isQuantityRemainderInDosesDisabled()"
+        placeholder="{{'openlmisInputDosesPacks.Doses' | message }}"
+        positive-integer />
+    <p> doses ) </p>
```

Comment:
Most likely we would like to have `doses` translated?

Reply:
added